src/components/AccountForms/LoginForm/index.js

```diff
@@ -0,0 +1,175 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { useForm } from 'react-hook-form';
+import { API_PATH } from 'utils/constants';
+import request from 'utils/axiosConfig';
+import { useAuth } from '../../../context/auth';
+import { GlobalContext } from 'context/global';
+
+import { Link, useHistory } from 'react-router-dom';
+import useStyles from './LoginForm.styles';
+import {
+  Paper,
+  TextField,
+  InputAdornment,
+  IconButton,
+  Button,
+} from '@material-ui/core';
+import { Visibility, VisibilityOff, Error } from '@material-ui/icons';
+import { ReactComponent as LoginHero } from './assets/LoginHero.svg';
+import { ReactComponent as TwitterLogo } from './assets/TwitterLogo.svg';
+
+const FORM_TITLE = 'Welcome back!';
+
+const emailRequirements = {
+  required: true,
+  pattern: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
```

Comment:
### Change to:

`required: 'Email is required',
  pattern: {
    value: /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
    message: 'Invalid email format',
  },`